### ISSUE TITLE
Resolve datahub issues

### ISSUE THREAD

**Issue by rmadupuri** [knowledgesystems/curation-scrum#510]
- [x] https://github.com/cBioPortal/datahub/issues/1211 yichao (resolved)
- [x] https://github.com/cBioPortal/datahub/issues/1133 yichao (review)
- [x] https://github.com/cBioPortal/datahub/issues/1069 ramya (waiting on timeline data from pedram)
- [x] https://github.com/cBioPortal/datahub/issues/1064 anusha (updated metadata, need to add additional files )
- [x] https://github.com/cBioPortal/datahub/issues/465 yichao (closed, part of another issue)
